apps/glusterfs/brick_entry.go

```diff
@@ -196,7 +196,6 @@ func (b *BrickEntry) Destroy(db wdb.RODB, executor executors.Executor) error {
 	req.Size = b.Info.Size
 	req.TpSize = b.TpSize
 	req.VgId = b.Info.DeviceId
-	req.Path = utils.BrickMountPoint(req.VgId, req.Name)
```

Comment:
I have a suggestion. Instead, make the destroy and destroycheck functions *only* refer to what is in Brick.Info. 
This is vital for my next patch, which changes the pattern of brickPath. In setups with existing bricks, only the new bricks would have VolumeName in the brickPath.

Reply:
I take this back. Everything looks good here.